content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,45 @@
+
+Indeed, sometimes the governing party of a single state might be tempted to gain an advantage by breaking a treaty. However, a single state won't have enough power to cause a national government to make these sorts of bad decisions, and justice will be preserved. See the peace treaty with Britain for an example of how important this is. <!-- Worth looking into what he's talking about -->
+
+These sorts of temptations often arise because a large number of residents of a state could benefit from them. Thus, they can be hard for the state government to resist. A national government made up of diverse states with different types of citizens will avoid these temptations and will therefore do better at keeping its treaties.
+
+So, as far as intentional or accidental violation of treaties is a cause for war, a national government will be better at keeping treaties than several smaller governments would be, and will therefore make us safer.
+
+When it comes to wars from direct and unlawful attacks, it is at least as clear to me that a single national government is vastly more secure against this kind of danger.
+
+These kinds of wars are usually caused by the passions of a faction (e.g., one or two states). For example, none of the wars with the American Indians have come about because of aggressions by the federal government<sup>[2](#footnote2)</sup>. Rather, these hostilities have been the result of provocations by individual states who were either unwilling or unable to punish wrongdoers, causing the slaughter of many innocent American Indians.
+
+This dynamic is most dangerous on our borders with Spanish and British territories. These bordering states might be tempted to lash out after a quarrel or irritation, dragging us into war with one of these powerful nations. Nothing can help to avoid this danger more than a national government, where decisions can be made outside of the context of the passions of the moment.
+
+Not only will a national government give fewer just causes of war, but it will also be better able to settle wars when they do happen. States, like men, are always trying to justify their actions rather than acknowledging and correcting errors or offenses. The national government will not be affected by this pride, but will act in moderation and candor to identify means to avoid and escape from difficulties.
+
+In addition, being united gives us a much stronger bargaining position. For example, in 1865 Louis XIV made Genoa send their chief magistrate and four senators to France to grovel before him. The Genoans had to do it for the sake of peace, but can you imagine Louis XIV demanding (or receiving) this humiliation from a powerful opponent like Spain or Britain?
```

Comment:
This paragraph is great! It definitely makes more sense now than when I read it the first time.